res/http-export/configuration.toml

```diff
@@ -26,6 +26,12 @@
         persistOnError = "false"
         secretheadername = "" # This is the name used in the HTTP header and also used as the secret key
         secretpath = ""
+    [Writable.Pipeline.Functions.HTTPPutJSON]
+      [Writable.Pipeline.Functions.HTTPPutJSON.Parameters]
+        url = "http://"
```

Comment:
why this url only has protocol but no address?

Reply:
@jim-wang-intel , It is just a place holder for the user to change to their actual URL.

Reply:
got it, maybe add some comments to indicate that like " # Add your own url or modify this"